src/main/java/de/relimit/commons/markdown/blockelement/paragraph/Paragraph.java

```diff
@@ -27,7 +28,7 @@ public Paragraph(String text) {
 	}
 
 	@Override
-	public Fences getFences() {
+	public Fences getFences(MarkdownSerializationOptions options) {
```

Comment:
Why is there an argument that is not used in the method?

Reply:
It is used in `Heading` so unfortunately we need to have it everywhere.